taxcalc/tests/test_calculate.py

```diff
@@ -117,14 +118,21 @@ def test_make_Calculator_deepcopy():
 
 
 def test_make_Calculator_from_files(paramsfile):
-    calc = Calculator.from_files(paramsfile.name, tax_dta_path,
-                                 start_year=1991,
-                                 inflation_rates=irates)
+    with open(paramsfile.name) as pfile:
+        params = json.load(pfile)
+    ppo = Parameters(parameter_dict=params, start_year=1991,
```

Comment:
@talumbau, I'm glad you bring this up, and @martinholmer, it would probably be useful for you to review the entire discussion in #284 as background, rather than just the top post. Apologies for not suggesting this earlier.

It's actually a bit more complicated than what the discussion in #284 might imply, which I get to two paragraphs down. 

First, the core takeaway is that it has long been @feenberg and my view that some users might prefer to specify a reform by way of a description of the reformed law in its entirety (a JSON file) rather than by way of a description of the changes in law from baseline (a parameter_dict). The way that this would work in practice is that a user would create a planX calculator from a PPO using the default `params.json` and a planY calculator from a PPO using some `planY_params.json` and then the user would compare the planX and planY calculators.  

Here is the additional complexity: a smaller subset of users might not even want to use `params.json` at all  -- there have, after all, been serious disagreements in the past over what the appropriate baseline is. These users might want to create the *planX* calculator from a PPO with `params_mybaseline.json` rather than `params.json` and then either use a JSON file *or* a parameter_dict to create the PPO for the planY calculator. Following this train of thought for the first time, I see that if the user wants to specify a different baseline with a JSON file to create the PPO for the planX calculator and then use a dictionary of reforms to specify changes from that baseline for the PPO for the planY calculator, then it means that a user will want to specify *both* a JSON file and a parameter_dict in order to create the PPO for the planY calculator. I don't see why this complexity would be a problem but wanted to point it out since it isn't documented elsewhere.